### PR TITLE
Use github actions to automatically run flake8 and mypy on push

### PR DESCRIPTION
**Pull request by heicrd**
This is sort of blocked by Irrational-Encoding-Wizardry/vsutil#36 so we can at least depend on master rather than a PR branch for vsutil. Ideally wait we'd just wait for a new release, depending on when that will be.

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      - uses: actions/checkout@v2
+      - name: Setup Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install VapourSynth
+        uses: Irrational-Encoding-Wizardry/install-vapoursynth-action@v0.1
+        with:
+          version: 50
+          cache: true
```
Comment (FichteFoll) [inline .github/workflows/lint.yml:18]: This is the default, FWIW.

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      - name: Install dependencies
+        run: |
+          python3 -m pip install --upgrade pip
+          pip install -r requirements-dev.txt
+          pip install .
+          pip install -U git+https://github.com/Irrational-Encoding-Wizardry/vsutil.git@stux-switch-to-package#egg=vsutil
+      - name: Running flake8
+        run: flake8 lvsfunc --count --ignore=E501,W503,E226 --show-source --statistics
+      - name: Running mypy
+        run: mypy -p lvsfunc --show-column-numbers --show-error-codes --hide-error-context --no-error-summary
```
Comment (FichteFoll) [inline .github/workflows/lint.yml:28]: I suggest adding this configuration in config files inside the repo instead of in the CI commands, so they can easily be reproduced locally through just `flake8 lvsfunc` or linter integration tools.

File: lvsfunc/__init__.py [hunk 3]
```diff
@@ -5,6 +5,8 @@
     or reach out to me on Discord (LightArrowsEXE#0476)!
 """
 
+# flake8: noqa
```
Comment (FichteFoll) [inline lvsfunc/__init__.py:8]: Why is this line needed? Just curious.

Comment (heicrd) [reply lvsfunc/__init__.py:8]: the imports are technically unused
